packages/docs/docs/layout-utils/fill-text-box.md

```diff
@@ -0,0 +1,91 @@
+---
+image: /generated/articles-docs-layout-utils-fill-text-box.png
```

Comment:
To generate the image: Remove this line, then run `node render-cards.mjs` in packages/docs